.github/workflows/ci_standalone.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         eda_server_version:
-          - main
+          - 93ef155accc3013f82a4870569638e7e1eaf2adc
```

Comment:
I think it would be good to put a comment after saying what version it is so it is easy to track